insolar/bus/retryer.go

```diff
@@ -123,39 +119,64 @@ func (r *RetrySender) waitForPulseChange(ctx context.Context, lastPulse insolar.
 	}
 }
 
+type receivedErrorType int
+
+const (
+	receivedErrorTypeNotError receivedErrorType = iota
+	receivedErrorTypeRetryable
+	receivedErrorTypeNonRetryable
+)
+
 // tryReceive returns false if we get retryable error,
 // and true if reply was successfully received or client don't want anymore replies
-func tryReceive(ctx context.Context, reps <-chan *message.Message, done chan struct{}, receiver chan<- *message.Message) bool {
-	for {
-		select {
-		case <-done:
+func tryReceive(
+	ctx context.Context,
+	reps <-chan *message.Message,
+	done chan struct{},
+	receiver chan<- *message.Message,
+	responseCount uint,
+) bool {
+	for i := uint(0); i < responseCount; i++ {
+		log.Info(i, responseCount)
```

Comment:
Use `inslogger` here, and some text in log would be nice too

Reply:
you're right, it was debug output. forgot to delete it. thx.